training/distributed_training/pytorch/model_parallel/bert/smp_bert_tutorial.ipynb

```diff
@@ -53,9 +53,33 @@
    "cell_type": "markdown",
```

Comment:
<p>Do you need to upgrade for some reason? This can break the customer experience for subsequent notebooks, so if you must upgrade, you should rollback at the end of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
The upgrade is required to make the Sagemaker SDK use the latest DLC, which comes with SMP. I had a rollback step in here (based on your example) which I removed because the python upgrades require kernel restart and the "original_version" is wrong after the restart. Is there a way to workaround the kernel restart ?